packages/honeycomb-opentelemetry-web/test/global-errors-instrumentation.test.ts

```diff
@@ -60,4 +64,19 @@ describe('Global Errors Instrumentation Tests', () => {
       });
     });
   });
+
+  describe('_computeStackTrace', () => {
+    it('should return an empty object if error is undefined', () => {
+      expect(instr._computeStackTrace(undefined)).toEqual({});
+    });
+
+    it('should return an object with structured stack trace information', () => {
+      expect(instr._computeStackTrace(new Error('This is an error'))).toEqual({
+        'exception.structured_stacktrace.columns': expect.any(Array),
```

Comment:
I'm happy to have this as a follow up PR but it would be great to assert on an actual transformation of a stack track here instead of checking that this returns an array. We would have to mock the `Error.stack` property and with a sample stack trace and then check that the results are the line numbers, column numbers and urls that we expect.

Reply:
I noticed a weird quirk with `TraceKit`, mocking `Error.stack` resulted in some lines being missing for certain stacks. `Error.originalStack` works perfectly though.

Reply:
Gotcha, I'm gonna approve this as is and we can tackle these tests separately. Can you leave a TODO comment here for that?

Reply:
We can also do this through smoke tests!

Reply:
Adding the TODO now